packages/cli/src/lib/snapshot/vault.ts

```diff
@@ -0,0 +1,154 @@
+import PlatformClient, {
+  VaultEntryModel,
+  VaultValueType,
+  VaultVisibilityType,
+} from '@coveord/platform-client';
+import {CliUx} from '@oclif/core';
+import {yellow} from 'chalk';
+import {readJsonSync, rmSync, writeJsonSync} from 'fs-extra';
+import open from 'open';
+import {join} from 'path';
+import {cwd} from 'process';
+import dedent from 'ts-dedent';
+import {InvalidVaultEntry} from '../errors/vaultErrors';
+import {AuthenticatedClient} from '../platform/authenticatedClient';
+import {Snapshot} from './snapshot';
+import {VaultEntryAttributes} from './snapshotReporter';
+
+export class Vault {
+  private static defaultEntryValue = '';
+  public constructor(private organizationId: string) {}
+
+  public async createEntries(entries: VaultEntryAttributes[]) {
+    const client = await this.client();
+    const vaultEntryFilePath = join(cwd(), `${this.organizationId}-vault.json`);
```

Comment:
cwd or project root?

Reply:
cwd will always be equal to project root since the push and preview commands should be executed in the project root.